src/components/OpenSource/WhyOpenSource.js

```diff
@@ -1,13 +1,21 @@
 import React, { Fragment } from 'react'
 import styled from 'styled-components'
-
+import breakpoint from 'styled-components-breakpoint'
 import { Grid, Row, Col } from '../grid'
 import TitleAndList from '../Common/TitleAndList'
 import Subtitle from '../Typography/Subtitle'
 import LogoGrid from '../Common/LogoGrid'
 
 const StyledGrid = styled(Grid)`
-  padding-bottom: ${props => props.theme.space[6]};
+  padding-bottom: ${props => props.theme.space[4]};
```

Comment:
Maybe all the CSS as `${({ theme }) => theme.spacing[4]}` for consistency. It's not a must do.

Reply:
I was using the destructuring logic yesterday as I wanted to get another prop for the background colour in the section wrapper. I reckon using `${(props) => props.spacing....}` is still ok. happy to mix them as its total preference.